drivers/at86rf215/at86rf215_netdev.c

```diff
@@ -541,9 +568,67 @@ static int _set(netdev_t *netdev, netopt_t opt, const void *val, size_t len)
                 at86rf215_configure_legacy_OQPSK(dev, at86rf215_OQPSK_get_mode_legacy(dev));
                 res = sizeof(uint8_t);
                 break;
+            case IEEE802154_PHY_MR_OQPSK:
+                at86rf215_configure_OQPSK(dev,
+                                          at86rf215_OQPSK_get_chips(dev),
+                                          at86rf215_OQPSK_get_mode(dev));
+                res = sizeof(uint8_t);
+                break;
             default:
                 return -ENOTSUP;
             }
+            break;
+
+        case NETOPT_MR_OQPSK_CHIPS:
+            if (at86rf215_get_phy_mode(dev) != IEEE802154_PHY_MR_OQPSK) {
+                return -ENOTSUP;
+            }
+
+            uint8_t chips;
+            assert(len <= sizeof(uint16_t));
+            if (*((const uint16_t *)val) <= 150) {
+                chips = 0;
+            } else if (*((const uint16_t *)val) <= 500) {
+                chips = 1;
+            } else if (*((const uint16_t *)val) <= 1500) {
```

Comment:
Aren't the chips values 100,200,1000,2000?

Reply:
Yea I thought I might make this 'work' for arbitrary chip values and choose the next best fit.
While nice when you don't exactly remember which chip rates there are, this is probably not something we want.